src/main/java/uk/gov/hmcts/reform/professionalapi/controller/SuperController.java

```diff
@@ -430,14 +431,14 @@ public String getUserEmail(String email) {
         if (nonNull(servletRequestAttributes)) {
 
             HttpServletRequest request = servletRequestAttributes.getRequest();
-            if (nonNull(request.getHeader("UserEmail"))) {
+            if (nonNull(request.getHeader(USER_EMAIL))) {
 
-                userEmail = request.getHeader("UserEmail");
-                log.warn("** Setting user email on the header  ** referer - {} ", request.getHeader("Referer"));
+                userEmail = request.getHeader(USER_EMAIL);
```

Comment:
now these loggers can be removed since email param is already decommissioned

Reply:
Addressed here: https://github.com/hmcts/rd-professional-api/pull/1072/commits/18f45aeb31ca469e13ddad9ad078b12cd2a3db67